### ISSUE TITLE
Create Monero donation address

### ISSUE THREAD

**Issue by anonimal** [monero-project/kovri#43]
When we get closer to a release, if/when people wish to donate, perhaps we should consider creating a Monero donation address or at least a writeup directing people to donate to Monero. In the future, I also think we should consider allowing personal donations to developers (if that helps to speed up development).

**Comment by KanediasTheMaker** [comment]
+ Also a team on salt.bountysource.com

**Comment by anonimal** [comment]
Thanks for the tip @KanediasTheMaker. @fluffypony, has Monero ever had a Bountysource discussion?

**Comment by fluffypony** [comment]
@anonimal Salt is new, haven't really looked at that (although we already have moneromooo who has been funded on a month-to-month basis for the past 8 months through the Forum Funding System). We developed the FFS (Forum Funding System) specifically because bounties didn't work for us. We've had two things developed with bounties, and in both instances the people that developed it disappeared afterwards. We've realised that bounties tend to attract lowest-common-denominator, quickest-to-the-post developers who have no pride in their work, and move on to the next bounty with no thought given to maintaining and building on their initial work. The FFS has given us a lot more longevity, people get paid what they want to get paid for the work they do when they do it.
